app/screens/home/search/results/file_card.tsx

```diff
@@ -113,7 +114,7 @@ export default function FileCard({fileInfo}: Props) {
                 />
             </View>
             <View style={style.flexColumn}>
-                <Text style={style.nameText}>{fileInfo.name}</Text>
+                <Text style={style.nameText}>{trimmedFilename}</Text>
```

Comment:
This is way simpler by using the Text `numberOfLines` attribute. Using that the system will automatically will add the `...` after the name of the file, and will work with different sizes. It is not the same length 10 `W` (WWWWWWWWWW) than 10 `i` (iiiiiiiiii).

Reply:
Thanks for this suggestion!  I added the change.